primitives/phragmen/src/lib.rs

```diff
@@ -391,13 +639,13 @@ pub fn build_support_map<Balance, AccountId, FS, C, R>(
 ///
 /// No value is returned from the function and the `supports` parameter is updated.
 ///
-/// * `assignments`: exactly the same is the output of phragmen.
-/// * `supports`: mutable reference to s `SupportMap`. This parameter is updated.
-/// * `tolerance`: maximum difference that can occur before an early quite happens.
-/// * `iterations`: maximum number of iterations that will be processed.
-/// * `stake_of`: something that can return the stake stake of a particular candidate or voter.
+/// `assignments`: exactly the same is the output of phragmen.
+/// `supports`: mutable reference to s `SupportMap`. This parameter is updated.
+/// `tolerance`: maximum difference that can occur before an early quite happens.
+/// `iterations`: maximum number of iterations that will be processed.
+/// `stake_of`: something that can return the stake stake of a particular candidate or voter.
```

Comment:
the doc rendered by rustdoc will not go to newline if there is no empty line.

```suggestion
/// - `assignments`: exactly the same is the output of phragmen.
/// - `supports`: mutable reference to s `SupportMap`. This parameter is updated.
/// - `tolerance`: maximum difference that can occur before an early quite happens.
/// - `iterations`: maximum number of iterations that will be processed.
/// - `stake_of`: something that can return the stake stake of a particular candidate or voter.
```